site/docs/plugins/keyboard.md

```diff
@@ -1,6 +1,23 @@
-# Keyboards and Inline Keyboards (built-in)
+# Inline and Custom Keyboards (built-in)
```

Comment:
"Inline and Custom" or "Custom and Inline"? The order differs

Reply:
This is intentional. It is now consistent with the article.

Reply:
Maybe to be more precise: This heading has a different order than the text in the vuepress config.ts

Reply:
Thank you, fixed.